WebJobs.Script.proj

```diff
@@ -7,7 +7,7 @@
     <SkipStrongNamesXml>tools\SkipStrongNames.xml</SkipStrongNamesXml>
     <PublishPath Condition=" '$(PublishPath)' == '' ">bin</PublishPath>
     <BuildNumber Condition=" '$(BuildNumber)' == '' ">0</BuildNumber>
-    <SiteExtensionVersion>0.1.$(BuildNumber)</SiteExtensionVersion>
+    <SiteExtensionVersion>0.2.$(BuildNumber)</SiteExtensionVersion>
```

Comment:
This should be it. If the package zip and the folder in it have the appropriate version, you should be good.